services/README.adoc

```diff
@@ -54,38 +54,49 @@ service will be receiving semi-arbitrary blobs of JSON from instances.
-                                             | core                  |
-                                             | createdAt             |
-                                             | updatedAt             |
-                                             +-----------------------+
+
+                                                +------------------------+
+                                                | versions               |
+                                                +------------------------+
+                                                |  id                    |
+                                             +--+ *uuid                  |
+                                             |  |  manifest (JSON)       |
+                           +------------+    |  |  manifestSchemaVersion |
+                           | instances  |    |  |  checksum (unique)     |
+                           +------------+    |  |  createdAt             |
+                           |  id        |    |  |  updatedAt             |
+                           | *uuid      <---++  +------------------------+
+                           |  timezone  |   ^^
+                           |  flavor    |   ||
+  +----------------+   +---+  updateId  |   ||
+  | updates        |   |   |  createdAt |   ||
+  +----------------+   |   |  updatedAt |   ||
+  | id             <---+   +------------+   ||
+  | commit         |                        ||
+  | channel        |                        ||  +---------------------+
+  | manifest (JSON)|                        ||  | capabilities        |
+  | tainted        |                        ||  +---------------------+
+  | createdAt      |                        ||  |  id                 |
+  | updatedAt      |                        |+--+ *uuid               |
+  +----------------+                        |   |  capabilities (JSON)|
+                                            |   |  createdAt          |
+                                            |   |  updatedAt          |
+                                            |   +---------------------+
+                                            |
+                                            |
+                                            |
+                                            |   +-----------------+
+                                            |   | connections     |
+                                            |   +-----------------+
+                                            |   |  id             |
+                                            +---+ *uuid           |
+                                                |  lastConnected  |
+                                                |  createdAt      |
+                                                |  updatedAt      |
+                                                +-----------------+
 
 * denotes an index
+`id` fields are always considered primary keys
```

Comment:
I don't think it renders as you expected it using \`id` as it's in the global preformatted source block.